src/documents/SubcomponentDeclaration.md

```diff
@@ -163,9 +164,16 @@ to which it is injected has a path `child2.injectedComponent` in the final insta
 called an _injected subcomponent_ because it appears at a path other than the one where it was constructed. In memory, the
 JavaScript object references at path `concreteChild` and `child2.injectedComponent` will be identical.
 
+Injecting subcomponents can be useful in order to capture (or parameterise) variability in the relationship between two components, in
+order to expose it more cleanly to a third component. For example, a component `A` may be managed by a component `B` elsewhere in the tree,
```

Comment:
I'm not really sure what I can do to improve the explanation - I've filed https://issues.fluidproject.org/browse/FLUID-6084 to represent this work in case anyone can contribute. Perhaps @simonbates has an idea since I think I successfully explained a use case for injected components to him in the context of the Nexus last week.